src/mono/System.Private.CoreLib/src/System/Reflection/Emit/MethodBuilder.Mono.cs

```diff
@@ -363,7 +363,7 @@ internal void fixup()
                 // do not allow zero length method body on MS.NET 2.0 (and higher)
                 if (((ilgen == null) || (ilgen.ILOffset == 0)) && (code == null || code.Length == 0))
                     throw new InvalidOperationException(
-                                         string.Format("Method '{0}.{1}' does not have a method body.",
+                                         SR.Format("Method '{0}.{1}' does not have a method body.",
```

Comment:
This should either use `string.Format` or the message should be moved into `Strings.resx`.

Reply:
since its an exception shouldnt it be SR.format?

Reply:
The basic rule is "if the format string is in the resx, then the format call should use `SR.Format`." In this case, the format string is currently in the C#, so it's not following the rule.